spec/linter-elixirc-spec.js

```diff
@@ -1,16 +1,115 @@
 'use babel';
 
 import { join } from 'path';
+import { remove } from 'fs-extra';
 
 const validPath = join(__dirname, 'fixtures', 'valid.ex');
-const warningPath = join(__dirname, 'fixtures', 'proj', 'mix.exs');
+const warningPath = join(__dirname, 'fixtures', 'proj', 'lib', 'proj.ex');
 const errorMode1Path = join(__dirname, 'fixtures', 'error-mode1.ex');
 const errorMode2Path = join(__dirname, 'fixtures', 'error-mode2.ex');
+const exsFilePath = join(__dirname, 'fixtures', 'script.exs');
+
+const mixBuildDirectory = join(__dirname, 'fixtures', 'proj', '_build');
+remove(mixBuildDirectory);
 
 describe('The elixirc provider for Linter', () => {
-  const lint = require('../lib/init.js').provideLinter().lint;
+  describe('when using the standard configuration', () => {
+    let lint;
+
+    beforeEach(() => {
+      atom.config.set('linter-elixirc.forceElixirc', false);
+      lint = require('../lib/init.js').provideLinter().lint;
+      atom.workspace.destroyActivePaneItem();
+
+      waitsForPromise(() =>
+        Promise.all([
+          atom.packages.activatePackage('linter-elixirc'),
+          atom.packages.activatePackage('language-elixir'),
+        ]),
+      );
+    });
+
+    it('works with mode 1 errors', () => {
+      waitsForPromise(() =>
+        atom.workspace.open(errorMode1Path).then(editor => lint(editor)).then((messages) => {
+          expect(messages.length).toBe(1);
+          expect(messages[0].severity).toBe('error');
+          expect(messages[0].html).not.toBeDefined();
```

Comment:
Good question. This is what the specs already had, so I'll revisit this to figure out the original intent and determine if `.url` is an appropriate update, or if `.html` should just be removed.